guides/Running-Tests.md

```diff
@@ -0,0 +1,14 @@
+# Running Tests
```

Comment:
I think I disagree slightly, @jugglinmike. Part of writing a test is being able to run it and so it should be included in the tutorial, at least in a lightweight way. Given that, I think it might be appropriate to write a short explanation about how to run it, perhaps linking to the `test262-harness` for more details, but covering enough that test writers aren't expected to learn a whole second project or left scratching their heads. I think that's a balance we could achieve. What do you think?

Reply:
Agreed--running the tests is definitely central to the contribution experience. It's just that Test262 intentionally doesn't offer that service, and I'm not convinced it has a perspective or audience that warrants original content on the topic. The kind guide that @rwaldron has outlined would be just as valuable (and even more relevant) to folks who arrive from the test262-harness repository.

I'd be more comfortable with something minimal--a command to install it, a command to run it, and a link to a version of this guide provided by test262-harness.

Reply:
I'll gladly put this in Test262-Harness, but we could keep short blurb described [here](https://github.com/tc39/test262/pull/3406/files#r823011869)

Reply:
I would like to keep the blurb and whatever it takes for the experience not to be "Oh ho, you wanted to RUN the tests? well that's a WHOLE different kettle of fish." I think that's putting idealism over a welcoming experience. But luckily I think we've arrived at a resonable decision.